src/main/java/org/jabref/model/entry/identifier/DOI.java

```diff
@@ -115,12 +133,37 @@ public DOI(String doi) {
             if (shortDoiMatcher.find()) {
                 this.doi = shortDoiMatcher.group(1);
                 isShortDoi = true;
+                checkForAmbiguousShortDOI(); // can only happen in this case
             } else {
-                throw new IllegalArgumentException(trimmedDoi + " is not a valid DOI/Short DOI.");
+                // Shortcut DOI without the "10/" as in "doi.org/d8dn"
+                Matcher shortcutDoiMatcher = EXACT_SHORT_DOI_SHORTCUT.matcher(trimmedDoi);
+                if (shortcutDoiMatcher.find()) {
+                    this.doi = "10/"+shortcutDoiMatcher.group(1);
+                    isShortDoi = true;
+                } else {
+
+                    throw new IllegalArgumentException(trimmedDoi + " is not a valid DOI/Short DOI.");
+                }
             }
         }
     }
 
+    /**
+     * checks whether the DOI might as well be something else, like:
+     * "10/2012" or " 10:as23 ".
+     *
+     * sets the property "isAmbiguous" accordingly. ( NOT USED YET! )
```

Comment:
We have the convention that in general dead/unused code should be removed. Do you have any concrete plans/ideas where the `isAmbiguous` variable is needed?
If not, I would ask you to remove it.

Reply:
Hi Tobias,
Yes I wasn't sure whether to open a new issue or to discuss my idea here in the comments somewhere:
When dois are cleaned up, the fields _Notes_ and _url_ are scanned for dois and short dois. The way it is implemented now, a short doi like eg **10/2021** is interpreted as a short doi, when this might very well be something else, like a date... 
_checkForAmbiguousShortDOI()_ detects these ambiguous cases and sets the property _isAmbiguous_ accordingly.
I stopped there, because I am still trying to find out whether a short doi like _10/1234_ is even possible i.e. whether it can be fully numeric. I wrote an email to contact@doi.org, but haven't heard back yet...
Also, I wanted to open a discussion on what to do with such ambiguous cases. We could implement some _DOIvalidityChecker_ that checks whether the supposed doi can be found. But a) I am not sure what would be the most efficient way of doing this and b) I am worried about scaling issues when there are thousands of ambiguous dois... Maybe the checking should be optional via a tick-box in respective gui?

Reply:
That's a good point to be aware of. For the conversation from the Notes and Url fields, I would say we can be optimistic and put it in the doi field if it's look like an DOI. I guess it's very rare to have only a date in the notes field.

I like the idea of checking doi's for existence (as an integrity check). I'm not sure how to implement it effectively, without querying crossref's api for each doi separately.

Reply:
ok. the way I understand you, the integrity checking would be applied to all DOIs right? should i remove the unused method and property regarding ambiguous short DOIs then?

Reply:
yes, integrity checks are always applied to all entries; and some users have 10k...

For now I would say remove the code for ambiguous dois. The code is not lost, as we can go back to this PR to copy it once it's really needed.